### ISSUE TITLE
replace clarifai features with tensorflow/keras object detection

### ISSUE THREAD

**Issue by rbroc** [neuroscout/neuroscout#842]
Object detection features can be obtained through tensorflow/keras models [already implemented in pliers](https://github.com/PsychoinformaticsLab/pliers/blob/master/pliers/extractors/models.py), which are probably just as good as clarifai, fine-tunable and free.
We should probably validate them on a subset of our stimuli, and if (as expected) performance does not get worse, replace clarifai features with these.

**Comment by tyarkoni** [comment]
Agreed—the one potential hitch (though it may not be an issue in practice) is that the pretrained models require an input of fixed size, so we currently apply the `ImageResizingFilter` naively internally, which will usually introduce distortion to the image. I imagine the Clarifai models are also fixed size, but they might be smarter about how they go about the resizing.

If nothing else, we probably should experiment with the `maintain_aspect_ratio` argument to `ImageResizingFilter` (and possibly also  the `resample` argument) prior to feeding the images to the KerasApplication extractor. @kaczmarj, do you remember if you evaluated this at all? If so, do you have any qualitative impressions of whether distorting or padding images produces better results? No worries if not.
